### ISSUE TITLE
Most recent commit removes compatibility with Windows

### ISSUE THREAD

**Issue by WAUthethird** [AIDungeon/AIDungeon#35]
Due to the recent additions to install.sh, it is now impossible to use the program on Windows. Please update this program for universal compatibility.

**Comment by JamesHutchison** [comment]
Could you be more specific? Is this with the Ubuntu subsystem?

**Comment by nickwalton** [comment]
Probably the aria2c command. You're welcome to add different install script for windows that uses something else. aria2c is handy because it can download a single large file in parallel.

**Comment by p-groarke** [comment]
Using git bash (which should be installed with git) :

Download wget (https://eternallybored.org/misc/wget/), place in `C:\Program Files\Git\mingw64\bin`.
Download aria2c (https://aria2.github.io/), place in `C:\Program Files\Git\mingw64\bin`.
Run `./install.sh` in git bash. For me, the requirements fail, so I install them through my system pip.

Launch a windows command prompt.
`pip install -r requirements.txt`

That's where I'm at. Right now most downloads are 403, so TBD if there is more to do.

**Comment by Portaluke** [comment]
So, now that things are somewhat working again, I'm trying to figure out how to install the game locally on Windows (I even installed CUDA hoping I'd be able to understand what to do afterwards).
Unfortunately now I'm lost... after a stream of experimenting and learning from other sources, I got this:
![cmd_43s5LWxRpZ](https://user-images.githubusercontent.com/58648868/70409895-58887b00-1a1b-11ea-9f57-393c9c48cf6b.png)
so... if anyone is able to help me, I'd be very happy to play this locally so I don't have to use up resources re-downloading the models should I need to leave my computer.

**Comment by lanekelly** [comment]
@Portaluke looks like you don't have python/pip installed, which install.sh assumes you have.

The install.sh apt-get commands don't seem to be running either, which install aria2c and unzip packages that are used download the torrent. That's why the script fails.

**Comment by Portaluke** [comment]
Any idea as to why the apt-get commands aren't working?
(also, I realize now that I didn't even have python installed so... oops)

**Comment by lanekelly** [comment]
Wrote down the steps I used to get things working:
http://familiarcycle.net/aidungeon-on-windows.html

Basically I deconstructed the install.sh file and ran the steps individually. I don't have WSL installed, just used regular Windows 10.

**Comment by ben-bay** [comment]
@WAUthethird @lanekelly what is the status of this issue?

**Comment by WAUthethird** [comment]
@ben-bay install.sh can be skipped if the torrent is downloaded via a separate client and if the files are put in the proper directory. requirements.txt must be installed via a separate command, but it works otherwise. I currently have it running on a potato Dell prebuilt from 2004 with an Athlon single-core, that I upgraded with 16 gigabytes of memory just for kicks. It's dead slow, as you might imagine, but it proves that as long as the memory is there, the AI will run on virtually anything.

*I did have to recompile TF with some removed processor extensions to support the older processor, but it otherwise works fine.
